pkg/ipamd/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
do we want to commit this one

Reply:
yeah this is not needed. will remove it.